src/Main/utils.cpp

```diff
@@ -0,0 +1,49 @@
+#include "utils.h"
+
+utils::utils(/* args */)
+{
+}
+
+utils::~utils()
+{
+}
+
+float utils::correctTemperature(float value) {
+  if (value < -40) {
+    return -40;
+  }
+  if (value > 100) {
+    return 125;
+  }
+  return value;
+}
+
+float utils::correctHumidity(float value) {
+  if (value < 0) {
+    return 0;
+  }
+  if (value > 100) {
+    return 100;
+  }
+  return value;
+}
+
+int16_t utils::correctCO2(int16_t value) {
+  if (value < 0) {
+    return 0;
+  }
+  if (value > 10000) {
+    return 10000;
+  }
+  return value;
+}
+
+int utils::correctPMS(int value) {
+  if (value < 10) {
```

Comment:
Why not allow values below 10? My sensors regularly go below that. Maybe limit it to 0?

Reply:
@Byter09 Please check the issue #190 and add your comments at there.

Reply:
I mean I don't have to. Even that issue says 0 should be the lowest value for PM data.